src/components/Dropdown/index.js

```diff
@@ -28,16 +27,25 @@ export default class Dropdown extends PureComponent {
     lastTimestamp: 0,
   }
 
+  toggleRef = React.createRef()
   dropdownRef = React.createRef()
+  arrowRef = React.createRef()
+
+  componentDidMount () {
+    this.renderDropdown()
+  }
 
   componentWillUnmount () {
     document.body.classList.remove('mc-dropdown__body--open')
   }
 
+  componentDidUpdate (prevProps) {
+    if (prevProps.placement !== this.props.placement) {
+      this.renderDropdown()
```

Comment:
shouldn't we be destroying the old popper instance before creating a new one?

Reply:
i gave that a shot, but their destroy method seemed to just cause nothing to render anymore.  since placement almost never changes (really only in the docs) i wasn't super worried about doing my own cleanup since i presume the garbage collection will take care of abandoned nodes.

Reply:
that, or we could just not support placement change